hs_app_netCDF/tests/test_netcdf_metadata_user_zone.py

```diff
@@ -39,23 +37,21 @@ def setUp(self):
         # file in iRODS user zone space can be read with metadata extracted correctly, other
         # functionalities are done with the same common functions regardless of where the netCDF
         # file comes from, either from local disk or from a federated user zone
-        self.irods_storage = IrodsStorage('federated')
         irods_target_path = '/' + settings.HS_USER_IRODS_ZONE + '/home/' + self.user.username + '/'
```

Comment:
@pkdash That would be too restrictive. I can foresee the need to use this ```save_files_to_user_zone()``` for different irods target path in different federated zone. The way it is implemented now allows the calling routine to pass in the pair for source and target, which would allow the calling routine to save files to different federated zone than the default one and with a different name if needed.